plugins/catalog/api-report.md

```diff
@@ -393,10 +401,10 @@ export interface EntityLayoutProps {
   //
   // (undocumented)
   UNSTABLE_contextMenuOptions?: EntityContextMenuOptions;
-  // Warning: (ae-forgotten-export) The symbol "ExtraContextMenuItem" needs to be exported by the entry point index.d.ts
+  // Warning: (ae-forgotten-export) The symbol "ExtraContextMenuItem_2" needs to be exported by the entry point index.d.ts
```

Comment:
can you please explain the change here? why did you add _2?

Reply:
This is just an artifact of how these machine generated api reports get produced by the scripts. You can double check at the top of the file what import it moved from/to.